### ISSUE TITLE
Avoid "Foodsoft Error" for unknown URLs

### ISSUE THREAD

**Issue by paroga** [foodcoops/foodsoft#370]
I regularly get the following error because some user want to access an not existing file "/apple-touch-icon.png":
```
An ActionView::MissingTemplate occurred in home#index:

  Missing template home/index, application/index with {:locale=>[:de, :en], :formats=>[:png], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :coffee, :haml]}. Searched in:
  * "/XXXXX/foodsoft/app/views"
  * "/XXXXX/foodsoft/plugins/forum/app/views"
  * "/XXXXX/foodsoft/plugins/documents/app/views"
  * "/XXXXX/foodsoft/plugins/messages/app/views"
  * "/XXXXX/foodsoft/plugins/wiki/app/views"
  * "/XXXXX/bundler/gems/forem-XXX/app/views"
  * "/XXXXX/gems/twitter-bootstrap-rails-XXX/app/views"
  * "/XXXXX/gems/kaminari-XXX/app/views"

  app/controllers/application_controller.rb:208:in `set_currency'
```

**Comment by wvengen** [comment]
Good point. I guess we could setup a default logo image? Perhaps we could improve on our current :)

**Comment by paroga** [comment]
For me it would be already ok if we could serve a simple 404 and don't raise an error. :-)

**Comment by wvengen** [comment]
Hmm, I guess I missed the point ;) Let's do that, I'll have a look.

**Comment by wvengen** [comment]
Some options:
* http://jerodsanto.net/2014/06/a-step-by-step-guide-to-bulletproof-404s-on-rails/
* https://gist.github.com/gonzedge/1563416
* http://thepugautomatic.com/2014/08/404-with-rails-4/
* http://stackoverflow.com/questions/13242947/rescuing-errors-in-rails-3-2-with-config-exceptions-app-self-routes
* https://github.com/mirego/gaffe
* https://github.com/eric1234/better_exception_app
I'm experimenting with [Gaffe](https://github.com/mirego/gaffe) and it seems to work quite well.

**Comment by wvengen** [comment]
PR in #375, would that solve your issue?

**Comment by wvengen** [comment]
Should be fixed with #375, please reopen when this is still a problem.
